src/proto/gps.proto

```diff
@@ -23,7 +23,8 @@ enum SampleType {
   IMAGE_FEAT = 16;
   END_EFFECTOR_POINTS_NO_TARGET = 17;
   END_EFFECTOR_POINT_VELOCITIES_NO_TARGET = 18;
-  TOTAL_DATA_TYPES = 19;
+  INDICATOR = 19;
```

Comment:
I don't think we should add it to the public repo unless it is used somewhere.

Chelsea